distribution/lib/Standard/Base/0.0.0-dev/src/Data/Json.enso

```diff
@@ -286,8 +287,7 @@ foreign js get_value object key = """
     return object[key]
 
 foreign js set_value object key value = """
-    object[key] = value
-    return object
+    return {...object, [key]: value}
```

Comment:
My intuition would tell me to write 
```suggestion
    return {...object, key: value}
```

Could you tell me what is the purpose of `[_]` here?

Reply:
otherwise JS treats key as the name of the field. The `[_]` makes it read the value of the variable.

Reply:
Ah! thanks for the explanation

Reply:
I think I often confuse the object syntax between JS and Python in how they handle variables there.